container_main.py

```
@@ -66,19 +66,34 @@ def run_scheduler():
+        cmd = [
+            "gunicorn",
+            "--bind",
+            f"0.0.0.0:{port_str}",  # nosec B104
+            "--workers",
+            "2",
+            "--timeout",
+            "120",
+            "api_server:app",
+        ]
+
+        logger.info(f"Launching gunicorn: {' '.join(cmd)}")
```

Ruff (0.15.1)

[warning] Logging statement uses f-string

(G004)